generator/generator.sh

```diff
@@ -31,7 +31,7 @@ fi
 mkdir k8s
 #generate kubernetes kubernetes manifest files
 echo "generating kubernetes manifest files"
-go run main.go generate $1 $2 $3
+go run main.go generate $1 $2
```

Comment:
why are we reducing the number of parameters? I think the three of them are still relevant...

Reply:
Ok, I think is because now we won't have a chain and a cluster file but instead only one file, is it?

Reply:
In the previous version, we had two arguments for files that we needed and one for readiness probe. In this PR we just need one file as input and the readiness probe, so the input parameters is reduced to 2.